packages/@webex/webex-core/test/unit/spec/services/services.js

```diff
@@ -72,6 +139,12 @@ describe('webex-core', () => {
       });
     });
 
+    describe('#initFailed', () => {
```

Comment:
we're missing a test that `initFailed===false` when initialization is successful

Reply:
added the test